rio/src/sequencer.rs

```diff
@@ -331,6 +332,10 @@ impl Sequencer {
                                     route.window.winit_window.select_previous_tab();
                                 }
                             }
+                            #[cfg(target_os = "macos")]
```

Comment:
I think hide is universal (besides wayland) no?

Reply:
Technically would not need the target_os

Reply:
key binding for this is only for MacOS and underline function (hide_application()) also implement for Macos only
If remove target_os filter, I think it still can run normally due to no key bind to call this

Reply:
Ah I see, I realised now that you are using hide_application, I thought hide also would work with https://docs.rs/winit/latest/winit/window/struct.Window.html#method.set_visible set_visible (that works for almost all platforms)

Reply:
Maybe hide can be global and if isn’t macos then use set_visible instead

Reply:
@raphamorim just try set_visible in MacOS, it's different with hide function that expect:
- hide: can back by alt + tab or icon on dock
- set_visible: don't how to back, maybe need call set_visible(true) to take it back

If you want I can add set_visible for other platform, so someone can bind on this

Reply:
Hmmm i see, well let’s implement only macos then and later we can revist it